worlds/stardew_valley/test/content/mods/TestSVE.py

```diff
@@ -60,10 +60,9 @@ def test_sve_has_sve_fish(self):
         self.assertIn(SVEFish.undeadfish, fish_names)
         self.assertIn(SVEFish.void_eel, fish_names)
         self.assertIn(SVEFish.water_grub, fish_names)
-        self.assertIn(SVEFish.dulse_seaweed, fish_names)
 
         # 63 pelican town  + 17 sve pelican town
-        self.assertEqual(63 + 17, len(self.content.fishes))
+        self.assertEqual(63 + 16, len(self.content.fishes))
```

Comment:
If you changed the 17 to 16, edit the comment too plz.

And if you want to go one step further, both the 63 and the 17 could be named variables/constants and remove the need for the comment altogether. No comment to update, no forgetting to update it :D